src/main/java/com/beerjournal/infrastructure/config/ApplicationProperties.java

```diff
@@ -0,0 +1,15 @@
+package com.beerjournal.infrastructure.config;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+
+@Component
+public class ApplicationProperties {
```

Comment:
I don't know if we need to create this component. (again, I know you took example from my code :P). I mentioned in previous comment what you can do with that ;)